README.md

```diff
@@ -1,21 +1,21 @@
-<p align="center"><a href="https://www.prisma.io"><img src="https://user-images.githubusercontent.com/7689783/59530137-1c001a80-8ee3-11e9-9463-aac1af43fcce.png" alt="Prisma Studio" height="66px"></a></p>
+<p align="center"><img src="https://github.com/prisma/studio/blob/readme_update/Studio.png" alt="Prisma Studio"></p>
```

Comment:
This is high risk as the branch will most certainly be deleted after the PR is merged. You should be able to able to upload it in a issue comment though and use that URL.

Reply:
Ouch, to late @madebysid

Reply:
I've changed that haha

Reply:
wow didn't even notice the URL, TIL